### PR TITLE
[UI] add ingressClassName

### PR DESCRIPTION
**Pull request by Mattes83**
This adds the ingressclass field to the ingress spec, see #1786
Signed-off-by: Matthias Teich <matthias.teich@gdata.de>

### CODE REVIEW

File: ui/manifests/ingress.yaml [hunk 1]
```diff
@@ -6,6 +6,7 @@ metadata:
   labels:
     application: "postgres-operator-ui"
 spec:
+  ingressClassName: "ingress-nginx"
```
Comment (FxKu) [inline ui/manifests/ingress.yaml:9]:
```suggestion
  # ingressClassName: "ingress-nginx"
```
better we only comment it

**Comment by FxKu** [comment]
:+1:

**Comment by jopadi** [comment]
👍

**Comment by FxKu** [comment]
@Mattes83 Thanks for your contribution!

**Pull request merged**